### PR TITLE
Neeva AI: Designs Part 2

### LINKED ISSUE #1249: WelcomeFlow: Actually center the content in LoadingScreen
Currently it is a set offset from the top of the screen. 

This won't work for all screen sizes.

### PR DESCRIPTION
**Pull request by Kokobe**
Adds Neeva AI designs in `CreateAccountScreen` and `PlansScreen`

### Pointers to help with review
* https://www.figma.com/file/6BWVIuiRpXepCbdZvlKK16/First-run---w-NeevaAI-messaging?node-id=3%3A2&t=dC65PlKV1TeJ75yP-0

### How was this tested?
- [x] I tested this manually on `debug`
- [ ] I tested this manually on `staging` or `release`
- [x] I added automated tests
- [ ] Existing automated tests cover some of my changes
- [ ] Existing automated tests cover all of my changes
- [x] I checked that `@Previews` still work in Android Studio

### Screenshots and screen recordings
Screen Shot 2023-02-23 at 9 53 54 AM

Screen Shot 2023-02-23 at 10 11 42 AM

### Issues Referenced
* Fixes #1249

### CODE REVIEW

File: app/src/main/java/com/neeva/app/welcomeflow/PlansScreen.kt [hunk 1]
```diff
@@ -325,35 +313,29 @@ private fun SubscriptionInfo(
...
-        )
-    ) { annotatedString, offset ->
-        annotatedString.getStringAnnotations(
-            tag = climatePledgeURL,
-            start = offset,
-            end = offset
+private fun PlansBenefit(title: String, description: String?) {
+    Row {
+        Text(
+            text = "•",
```
Comment (dan-alcantara-neeva) [inline app/src/main/java/com/neeva/app/welcomeflow/PlansScreen.kt:319]: I think we technically have this in the strings.xml file already.  It's "dot" or something?

File: app/src/main/java/com/neeva/app/welcomeflow/PlansScreen.kt [hunk 2]
```diff
@@ -325,35 +313,29 @@ private fun SubscriptionInfo(
...
+            style = MaterialTheme.typography.titleLarge,
+            color = MaterialTheme.colorScheme.primary,
+            modifier = Modifier.padding(horizontal = Dimensions.PADDING_MEDIUM)
         )
-            .firstOrNull()
-            ?.let {
-                openURL(Uri.parse(it.item))
-                return@AnnotatedSpannable
+        Column {
+            Spacer(modifier = Modifier.size(4.dp))
```
Comment (dan-alcantara-neeva) [inline app/src/main/java/com/neeva/app/welcomeflow/PlansScreen.kt:325]: Dimensions.PADDING_TINY?

**Review by dan-alcantara-neeva — CHANGES_REQUESTED** [review]
You'll need to figure out what to do about short screens:
screenshotTests/src/main/assets/golden/com.neeva.app.welcomeflow.WelcomeFlowScreenshotTests_createAccountScreen_OtherSignInOptions_Light_Preview.png

All this random text has pushed the actually useful "sign in" button off of the screen with no indication that it's there.

**Comment by hwiatneeva** [comment]
- column 2: i have no solution of one design to cover the all cases. it needs a special layout in this size. 
- column 4: button text needs to be center-align, unless this language is more proper for left-aligned.

**Comment by Kokobe** [comment]
> * column 2: i have no solution of one design to cover the all cases. it needs a special layout in this size.
> * column 4: button text needs to be center-align, unless this language is more proper for left-aligned.

Fixed! Updated the PR description

**Review by dan-alcantara-neeva — APPROVED** [review]

**Pull request merged**